### ISSUE TITLE
Elements should not have redundant roles.

### ISSUE THREAD

**Issue by dengeist** [morgannewman/womby#6]
**Affected component:** `NoteCard`. 

The browser console warning explains this nicely: 
> The element `button` has an implicit role of button. Defining this explicitly is redundant and should be avoided.
